README.md

```diff
@@ -207,6 +207,12 @@ It is also common in the Node.js ecosystem to import files without including the
 
 would allow not only `import fp from "lodash/fp.js"`, but also allow `import fp from "loadsh/fp"`.
 
+Although this example shows how it is _possible_ to allow extension-less imports with import maps, it's not necessarily _desirable_. Doing so bloats the import map, and makes the package's interface less simple—both for humans and for tooling.
+
+This bloat is especially problematic if you need to allow extension-less imports within a package. In that case you will need an import map entry for every file in the package, not just the top-level entry points. For example, to allow `import "./fp"` from within the `/node_modules/lodash-es/lodash.js` file, you would need an import entry mapping `/node_modules/lodash-es/fp` to `/node_modules/lodash-es/fp.js`. Now imagine repeating this for every file referenced without an extension.
```

Comment:
> For example, to allow `import "./fp"` from within the `/node_modules/lodash-es/lodash.js` file, you would need an import entry mapping `/node_modules/lodash-es/fp` to `/node_modules/lodash-es/fp.js`.

This currently reads like import maps apply after URL resolution. Is that correct? My understanding was that they apply *before* URL resolution..? Example:

* `"/foo/bar": "/foo/zapp"` - for the exact specifier "/foo/bar", replace with "/foo/zapp", then resolve URL based on base to `https://origin.com/foo/zapp`.

Or is the import map tried multiple times, once on the original string and once on the URL after resolution relative to the base? If it is only looked at before resolution relative to the base/referrer, then shouldn't `"./fp"` require its own unique entry in the import map and wouldn't be caught by `"/node_modules/lodash-es/fp"`?

Reply:
If a specifier is URL-like, e.g. starts with ./ or ../, then URL resolution applies. See #112 or the reference implementation for more details.